ci-operator/config/openshift/linuxptp-daemon/openshift-linuxptp-daemon-master__okd-scos.yaml

```
@@ -0,0 +1,45 @@
+base_images:
+  ocp_4.16_base-rhel9:
+    name: "4.16"
+    namespace: ocp
+    tag: base-rhel9
+  ocp_builder_rhel-9-golang-1.21-openshift-4.16:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.21-openshift-4.16
+  origin_scos-4.16_base-stream9:
+    name: scos-4.16
+    namespace: origin
+    tag: base-stream9
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: Dockerfile
+  inputs:
+    ocp_builder_rhel-9-golang-1.21-openshift-4.16:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16
```

Please include image alias as well.
```suggestion
      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16
      - builder
```
This helps the replacement rule continue to work even if the there is a golang bump.

---

updated